index.md

```diff
@@ -53,7 +53,7 @@ Thursday:
 	* [RNAseq and differential expression](https://angus.readthedocs.io/en/2018/rna-seq.html)
 
 Friday:
-* 9am: morning lecture: Amanda, on statistics and experimental design
+* 9am: morning lecture: Amanda, on statistics and experimental design [Slides](https://github.com/ngs-docs/angus/blob/2018/StatsNstuff.pdf)
```

Comment:
@karenword Could you change this to (https://github.com/ngs-docs/angus/raw/2018/StatsNstuff.pdf). It should make it easier to download.